xml/System.Net.Security/CipherSuitesPolicy.xml

```diff
@@ -39,9 +39,19 @@
         <Parameter Name="allowedCipherSuites" Type="System.Collections.Generic.IEnumerable&lt;System.Net.Security.TlsCipherSuite&gt;" />
       </Parameters>
       <Docs>
-        <param name="allowedCipherSuites">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="allowedCipherSuites">The collection of cipher suites allowed in this policy for negotiation.</param>
+        <summary>Initializes a cipher suite policy that can be used on a client or server authentication.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+> [!NOTE]
+> Defining a chipher suite policy on <xref:System.Net.Security.SslStream> authentication will prevent the OS of the ability to decide which are the best cipher suites to negotiate with and will need of your attention to manually check and update this code. It is strongly recommended to hold off the use of this property and rely on your constantly updated OS policy.
+
+## Remarks
+Unlike the OS underlying SSL/TLS implementation, the order of the <xref:System.Net.Security.TlsCipherSuite> elements in `allowedCipherSuites` does not determine their priority in the client-server negotiation. It is required to use OSX or Unix-based systems with OpenSSL 1.1.1 in order to initialize an instance of this class.
```

Comment:
Does it actually not determine the order?  I thought it did on macOS and Linux (but on Windows it won't).

Reply:
on OSX I think it does, on Linux it will do some re-ordering (TLS1.3 cipher suites will go first, possibly some null ciphers will go to the back - can't remember the details) but mostly preserves the order. I'd just keep the sentence to `order is not guaranteed` (or we will do best effort or something along these lines)